build-tools/xaprepare/xaprepare/ThirdPartyNotices/libzip.cs

```diff
@@ -18,5 +18,15 @@ class libzip_nih_at_libzip_TPN : ThirdPartyNotice
 		public override string LicenseText => null;
 
 		public override bool Include (bool includeExternalDeps, bool includeBuildDeps) => includeExternalDeps;
+
+		private static string GetNugetLocation()
```

Comment:
Could we move this helper method into https://github.com/xamarin/xamarin-android/blob/master/build-tools/xaprepare/xaprepare/Application/Context.cs or maybe https://github.com/xamarin/xamarin-android/blob/master/build-tools/xaprepare/xaprepare/ConfigAndData/Configurables.cs so that it can be called by all places looking for a NuGet package path, rather than duplicating this logic across three classes?